code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2003,3 +2003,30 @@
 	color = "#ED2939"
 	taste_description = "upside down"
 	can_synth = FALSE
+
+/datum/reagent/consumable/ratlight
+	name = "Ratvarian Light"
+	description = "A special concoction said to have been blessed by an ancient god. Makes the consumer glow with literal enlightenment."
+	color = "#B5A642"
+	taste_description = "enlightenment"
+	metabolization_rate = 0.8 * REAGENTS_METABOLISM
+	var/datum/language_holder/prev_language
+
+/datum/reagent/consumable/ratlight/reaction_mob(mob/living/M)
+	M.set_light(2)
+	..()
+
+/datum/reagent/consumable/ratlight/on_mob_life(mob/living/carbon/M)
+	playsound(M, "scripture_tier_up", 50, 1)
```

Comment:
This will play a lot

Reply:
Luckily I made the code and I don't think it works unless he fixed it. If he did then I do agree